src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/ViewModels/MappingsViewModel.cs

```diff
@@ -1,6 +1,6 @@
 namespace OrchardCore.Search.Elasticsearch.ViewModels
 {
-    internal class MappingsViewModel
+    public class MappingsViewModel
```

Comment:
@Skrypt is there any reason that you made it `internal`?

Reply:
No reason to be internal. Probably automatic code added and I did not double-check because I did add this one pretty quickly.